tensorflow/python/ops/ctc_ops.py

```diff
@@ -149,7 +150,8 @@ def ctc_loss(labels, inputs, sequence_length,
       sequence_length,
       preprocess_collapse_repeated=preprocess_collapse_repeated,
       ctc_merge_repeated=ctc_merge_repeated,
-      ignore_longer_outputs_than_inputs=ignore_longer_outputs_than_inputs)
+      ignore_longer_outputs_than_inputs=ignore_longer_outputs_than_inputs,
+      name=name)
```

Comment:
Can you add a unit test to test that using `ctc_loss` with the new `name` kwarg works.

Reply:
ok, i will do this as soon as possible

Reply:
done